articles/protocols/saml/adfs.md

```diff
@@ -4,202 +4,139 @@ toc: true
-
-| LDAP Attribute | Outgoing Claim |
-| - | - |
-| E-Mail-Addresses | E-Mail Address |
-| Display-Name | Name |
-| User-Principal-Name | Name ID |
-| Given-Name | Given Name |
-| Surname | Surname |
+After you close the **Add Relying Party Trust** wizard, the **Edit Claim Issuance Policy** window appears.
 
-The only mandatory mapping you need is for the email address, but we strongly recommend adding all of the ones listed above, especially **Name ID**, since they are the ones most commonly used.
+1. Click **Add Rule...** to launch the wizard. 
+2. Select **Send LDAP Attributes as Claims** for your **Claim rule template**, and click **Next**.
+3. Provide a value for the **Claim rule name**, such as "LDAP Attributes" (it can be anything you want). 
+4. Choose **Active Directory** as your **Attribute Store**. 
+5. Map your LDAP attributes to the following outgoing claim types:
 
-Click **Finish**.
+    | LDAP Attribute | Outgoing Claim |
+    | - | - |
+    | E-Mail-Addresses | E-Mail Address |
+    | Display-Name | Name |
+    | User-Principal-Name | Name ID |
+    | Given-Name | Given Name |
+    | Surname | Surname |
 
-![](/media/articles/protocols/saml-adfs/saml13.png)
+    ::: note
+    The `Name ID` outgoing claim should always be present to ensure correct session handling. We strongly recommend adding all of the claims listed above, especially `E-Mail Address`, since they are the ones most commonly used.
+    :::
 
-Back on the **Edit Claim Issuance Policy** window, click **Apply**. 
+    You can add additional claim mappings if necessary. See [Connect Your Application to Microsoft ADFS](/connections/enterprise/adfs#add-additional-ldap-attributes) for details.
 
-![](/media/articles/protocols/saml-adfs/saml14.png)
-
-You can now exit out of this window.
+6. Click **Finish**.
+7. In the **Edit Claim Issuance Policy** window, click **Apply**. You can now exit out of this window.
 
 ### Obtain the Federation Metadata
 
-To get your Federation Metadata, navigate to the following URL:
-
-`https://your-server/FederationMetadata/2007-06/FederationMetadata.xml`
-
-Your file will looking something like this:
-
-![](/media/articles/protocols/saml-adfs/saml21.png)
-
-Save the file for later use.
-
-### Export the Signing Certificate
-
-Finally, you'll need to export the signing certificate from the ADFS console to upload it to Auth0 at a later point.
-
-Using the left-hand navigation pane, go to **ADFS > Service > Certificates**. Select the **Token-signing** certificate, and right click to select **View Certificate**.
-
-![](/media/articles/protocols/saml-adfs/saml15.png)
-
-On the **Details** tab, click **Copy to File...**.
-
-![](/media/articles/protocols/saml-adfs/saml16.png)
-
-This launches the Certificate Export Wizard. Click **Next** to proceed.
+1. To get your Federation Metadata, navigate to the following URL:
```

Comment:
I believe that the whole "Obtain the federation metadata" step can be omitted. The sole purpose is to obtain the callback URL (ACS URL), which is always in the form of `https://{auth0_domain}/login/callback?connection={connection_name}`. This format is explained below, so no need to do this step.

Reply:
Deleted